renovate.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```

Comment:
The Dependabot config provided a schedule that limited how frequently PRs were created. Can you update the Renovate configuration to retain that schedule?